include/deal.II/sundials/arkode.h

```diff
@@ -541,6 +607,33 @@ namespace SUNDIALS
     void
     reset(const double t, const double h, const VectorType &y);
 
+    /*!
+     * Create a new SUNDIALS vector from a given template.
+     *
+     * TODO: this method really doesn't belong here and should be removed once
+     * N_Vector "understands" our vectors
+     *
+     * @param template_vector The vector to use as a template for the layout of
+     * a new vector.
+     */
+    N_Vector
+    create_vector(const VectorType &template_vector) const;
```

Comment:
See my comment below. This is only a pointer. And yes only used internally, but it needs the communicator in case of a parallel run. I left it here because i suggest to remove the `copy` methods between vectors with a second PR.